.spelling

```diff
@@ -149,6 +150,7 @@ AuthorizationPolicy
 authorizer
 AuthPolicy
 authz
+auto-san-validation
```

Comment:
Please remove unnecessary additions to .spelling. Most of these should be escaped with back-ticks in the doc, not added here.